src/mmw/js/src/core/layerPicker.js

```diff
@@ -186,6 +276,20 @@ var LayerPickerGroupView = Marionette.LayoutView.extend({
             layerGroupName: this.model.get('name'),
             message: message,
         };
+    },
+
+    showTimeSliderRegion: function(timeSlider) {
+        if (!this.timeSliderRegion) { return; }
+
+        this.timeSliderRegion.show(timeSlider);
+        this.timeSliderRegion.$el.show();
+    },
+
+    hideTimeSliderRegion: function() {
+        if (!this.timeSliderRegion) { return; }
+
+        this.timeSliderRegion.empty();
+        this.timeSliderRegion.$el.hide();
```

Comment:
Consider adding an `:empty` css selector in sass for `#layer-picker-slider-region` instead.